code/handlers/trackservers.q

```diff
@@ -334,11 +334,12 @@ reqprocsnotconn:{[requiredprocs]
   }
 
 // Block process until all required processes are connected
-startupdependent:{[requiredprocs;timeintv]
-    while[.servers.reqprocsnotconn[requiredprocs];
-          .os.sleep[timeintv];
-          .servers.startup[]
-         ]
+startupdependent:{[requiredprocs;timeintv;cycles;subscribe;process]
+     n:();  //variable used to check how many cycles have passed                                                                                                                                                 
```

Comment:
Should this be `a` and should it be set to zero? I can't see it being used again in this function.